docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst

```diff
@@ -0,0 +1,39 @@
+====================
+Ansible project 2.10
+====================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-base <https://pypi.org/project/ansible-base/>` package as well. All dates are subject to change, please see :ref:`base_roadmap_2_10` for the most recent updates on ansible-base.
```

Comment:
```suggestion
This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-base <https://pypi.org/project/ansible-base/>`_ package as well. All dates are subject to change. See :ref:`base_roadmap_2_10` for the most recent updates on ``ansible-base``.
```

Reply:
Thanks @samccann, I rolled this into the other suggested changes in 69b44a960b2684a72fa25d2b477e46980fdf03d4.